docs/src/content/faq.mdx

```diff
@@ -48,27 +81,275 @@ After receiving the NFT, follow these steps to set up the Icebreaker platform:
+This is expected behavior and should not cause concern unless persistent issues occur.
+If so, check your logs for errors.
+
+## Maintenance and advanced topics
+
+### What is the process for updating my Icebreaker node to a new version of the Blockfrost platform?
+
+When a new version of the Blockfrost platform is released:
+
+1. Watch for an announcement in the private Icebreakers Discord channel with update instructions
+2. Shut down your current `blockfrost-platform` process
+3. Update the platform software using the same [installation method](installation/installers) you originally used
+4. Restart the `blockfrost-platform` process
+5. Verify the new version is running with `blockfrost-platform --version`
+6. Monitor your logs and the dashboard to ensure your node reconnects successfully.
+
+Promptly updating is recommended to maintain compatibility and access new features.
+The specific update steps may vary slightly between releases so always refer to the official release notes.
+
+### What are the rules around sharing my Icebreaker secret key?
+
+Your Icebreaker secret key is like a password for your node.
+It should be kept confidential and never shared publicly.
+The secret key authorizes your node to participate in the Icebreakers platform.
+
+Specific guidelines:
+
+1. Don't post your secret key on any public websites, forums, or social media
+2. Don't share the key with untrusted parties
+3. Only store the key on servers you control and trust
+4. Use a unique secret key for each Icebreaker node you run
+5. If you suspect your key is compromised, generate a new one and update your node.
+
+The Blockfrost team will never ask for your secret key.
+Be cautious of phishing attempts.
+
+If you lose your key, reach out to the team on Discord for assistance with generating a new one.
+
+### How can I improve the security of my Icebreaker node?
+
+For enhanced security:
+
+1. Limit access to the Blockfrost platform to specific IP addresses.
+2. Monitor updates on [GitHub issue #178](https://github.com/blockfrost/blockfrost-platform/issues/178) for NAT handling improvements.
+
+### How can I contribute to improving the Blockfrost platform?
+
+You can contribute by:
+
+1. Reporting bugs and suggesting features on the [Blockfrost platform GitHub repository](https://github.com/blockfrost/blockfrost-platform).
+2. Participating in community discussions on Discord to share feedback and ideas.
+
+### I found a typo or an error in the documentation. Where should I report it?
+
+Please report typos, errors, and any other documentation issues on the [Blockfrost platform GitHub repository](https://github.com/blockfrost/blockfrost-platform/issues).
+
+### What is the process for reporting a bug in the Blockfrost platform?
+
+Report bugs by:
+
+1. Creating an issue on the [Blockfrost platform GitHub repository](https://github.com/blockfrost/blockfrost-platform).
+2. Providing detailed information about the bug and steps to reproduce it.
```

Comment:
It would be nice to mention provide where?

Reply:
@ginnun  I added more detailed guidance about the process for creating an issue and where to provide details.